python/lsst/ip/isr/isrTask.py

```diff
@@ -610,8 +621,10 @@ def saturationDetection(self, exposure, amp):
         """
         if not math.isnan(amp.getSaturation()):
             maskedImage = exposure.getMaskedImage()
-            dataView = maskedImage.Factory(maskedImage, amp.getRawBBox())
-            isr.makeThresholdMask(
+            for box in (amp.getRawBBox(), amp.getRawHorizontalOverscanBBox(),\
+                        amp.getRawVerticalOverscanBBox()):
+                dataView = maskedImage.Factory(maskedImage, box)
+                isr.makeThresholdMask(
                 maskedImage = dataView,
```

Comment:
I was thinking you were talking about line 625/626, I see what you are saying now and they are now indented.